google/cloud/dataproc_v1/gapic/autoscaling_policy_service_client.py

```diff
@@ -88,15 +78,24 @@ def from_service_account_file(cls, filename, *args, **kwargs):
     from_service_account_json = from_service_account_file
 
     @classmethod
-    def autoscaling_policy_path(cls, project, region, autoscaling_policy):
+    def autoscaling_policy_path(cls, project, location, autoscaling_policy):
         """Return a fully-qualified autoscaling_policy string."""
         return google.api_core.path_template.expand(
-            "projects/{project}/regions/{region}/autoscalingPolicies/{autoscaling_policy}",
+            "projects/{project}/locations/{location}/autoscalingPolicies/{autoscaling_policy}",
```

Comment:
Check why this resource path helper method is changing

Reply:
@bradmiro Are `region` and `location` interchangeable?

The proto annotation specifies two kinds of patterns.
https://github.com/googleapis/googleapis/blob/7a8875d4bcf51993f6f7aa181066cc5ec95539d5/google/cloud/dataproc/v1/autoscaling_policies.proto#L106-L111

The older config only had `region`.
https://github.com/googleapis/googleapis/blob/7a8875d4bcf51993f6f7aa181066cc5ec95539d5/google/cloud/dataproc/v1/dataproc_gapic.legacy.yaml#L37-L38

Reply:
Luke says that `region` and `zone` were used by APIs before this was standardized to `location`. Given that the proto annotation specifies both paths this change should not impact users.